gettsim/benefits/wohngeld.py

```diff
@@ -195,20 +193,29 @@ def _calc_wg_income_deductions_until_2015(household, tb):
     """ calculate special deductions for handicapped, single parents
     and children who are working
     """
+    household["child_below_11_tu"] = household.groupby("tu_id")["age"].apply(
+        count_child_below_11_tu
+    )
```

Comment:
Remove ``count_child_below_11_tu``. Even if we are not concerned about performance for now, ``apply`` is essentially a for-loop and the function is not very clear. What about ``household.groupby("tu_id")["age"].transform(lambda x: x.le(11).sum())``. And should it be children instead of child?

Reply:
Yeah, I guess the variable should be called "n_children_below_11_tu"

Reply:
That looks a lot better. Adapted it!